packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -532,27 +533,27 @@ abstract class FlutterDriver {
   ///        `screenshot()`. The gap is random because it is determined by the
   ///        unpredictable network communication between the driver process and
   ///        the application. If this gap is too short, which it typically will
-  ///        be, the screenshot is taken before the GPU thread is done
+  ///        be, the screenshot is taken before the raster thread is done
```

Comment:
Nit: formerly known as the GPU thread

Reply:
This is already explained a few lines above. I think it's better to put the "formerly known" explanation only once per each doc comment.